app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def render_not_found
```

Comment:
@GabrielleEaston I moved this method to the ApplicationsController, so we can use it app-wide.

Let me know if you have questions.